### DISCUSSION TITLE
7ECBBF02-BC1F-5801-0CCA-59871B88B44A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#25] [General]
# SS 433

SS 433 is an eclipsing X-ray binary star system (consisting of a stellar-mass black hole and a late-stage A-type star) about 18,000 light-years away in the constellation Aquila. It is the first known microquasar ever found.

(NOTE: Black hole addons tend to be performance heavy and might not work in all devices)

https://celestia.mobi/addon?item=7ECBBF02-BC1F-5801-0CCA-59871B88B44A&lang=en

**Comment by Dirgham2015** [comment]
WoOoOw

**Comment by TayTay2014** [comment]
Cool

**Comment by 11azr** [comment]
سبحان الله

**Comment by Ksplover2343** [comment]
W

**Comment by Zayedeisa2016** [comment]
Ok

**Comment by Jonzxsy** [comment]
Mt bonito,recomendo olhar

**Comment by Alaxkibz** [comment]
Não gostei

**Comment by yurimarcus009-ctrl** [comment]
ꋊ꒐ꍌꍌꏂꋪ
